src/Models/ModelInstantiatorTest.php

```diff
@@ -21,17 +22,24 @@ public function testArrayToDataObject(): void
             'firstName'    => 'Peter',
 
             // testProperty exists in the class but does not have a Column attribute.
-            'testProperty' => 'test',
+            'testProperty' => true,
```

Comment:
Why the change here?

Reply:
Because the property is defined as a bool in the class which caused the validation to fail when it's set as a string here.

Reply:
All right. Apologies if we already discussed this; do we want to validate properties that don't have `Column` attributes?

Reply:
I'd say yes, since the function is converting an array to a data object based on a class definition and the attribute is defined on the class. This specific function doesn't seem to care about `Column` just attributes.

Reply:
`arrayToDataObject` ignores properties that don't have a `Column` attribute. It only assigns/converts properties that are returned by `ModelInstantiator#dataObjectPropertyColumns`